build-system/tasks/helpers.js

```diff
@@ -738,11 +730,9 @@ async function maybePrintCoverageMessage(covPath = '') {
  * various runtime files.
  *
  * @param {string} targetFile File to which the config is to be written.
- * @param {boolean} localDev Whether or not to enable local development.
- * @param {boolean} fortesting Whether or not to enable testing mode.
```

Comment:
This change SGTM based on what I know. But is there any remaining use case where `isProd` is false, and one of `localDev` or `fortesting` also needs to be false? I'm pretty sure the answer is no, but just checking.

Reply:
Within the bundled code, no. Within build-system code, I don't think so